ansible/roles/dashd/tasks/main.yml

```diff
@@ -56,6 +56,26 @@
     - docker-compose.yml
     - .env
 
+- name: Configure log rotation
+  include_role:
+    name: arillso.logrotate
+  vars:
+    logrotate_use_hourly_rotation: true
+    logrotate_applications:
+      - name: core-logs
+        definitions:
+          - logs:
+              - "{{ dashd_home }}/.dashcore/{{ 'devnet-' + dash_devnet_name if dash_network == 'devnet' else 'testnet3' }}/debug.log"
+            options:
+              - rotate 7
+              - daily
+              - maxsize 1G
```

Comment:
100M should be more than enough. 7GB just for logs (1GB * 7 log files) is a lot.

Reply:
True, but core in debug mode can generate a lot of logs on the miner when a bug such as `bad-qc-not-allowed` is encountered. We filled ~15GB of free space on the node in under 3-4h, so I think 1GB is a reasonable threshold if logrotate runs hourly. The main point of this is just to stop the drive from running out of space, the logs we want to keep will be safely stored in elasticsearch (after #325 is merged).

Reply:
What you need is a file like `/etc/logrotate.d/dashd` 
```
/dash/.dashcore/testnet3/debug.log {
        rotate 9
        copytruncate
        compress
        notifempty
        missingok
        size 1G
}
```
I have tested this and it creates good logs without missing any, I prefer compressed logs, use `zcat` and `zgrep` to view and search them.

Next edit the file `/usr/lib/systemd/system/logrotate.timer` and change the word `daily` to `hourly`.
```
OnCalendar=hourly
```
Then either reboot (preferred) or daemon-reload.